src/views/Accessibility/AccessibilityRequestDetailPage.tsx

```diff
@@ -355,9 +350,12 @@ const AccessibilityRequestDetailPage = () => {
               <>
                 <Form className="usa-form maxw-full">
                   <FieldGroup>
-                    <Label htmlFor="CreateAccessibilityRequestNote-NoteText">
+                    <label
```

Comment:
Why is this changed?

Reply:
I needed to avoid the `margin-top` property in `usa-label`

Reply:
What do you mean by avoid? I'm not seeing a margin top here with the `<Label />` component.

<img width="1910" alt="Screen Shot 2021-07-06 at 12 02 42 PM" src="https://user-images.githubusercontent.com/8367504/124653506-3acb0400-de52-11eb-8d03-3f753583acf6.png">

Reply:
I think we want the Note label to line up with the blue bar above Test Dates and Scores

Reply:
I think you'd need to remove the margin on the `FieldGroup`.

Reply:
wait

Reply:
for some reason my node_modules hadn't updated to include that